app/src/main/kotlin/at/bitfire/davdroid/settings/AccountSettingsMigrations.kt

```diff
@@ -58,9 +67,38 @@ class AccountSettingsMigrations @AssistedInject constructor(
         fun create(account: Account, accountSettings: AccountSettings): AccountSettingsMigrations
     }
 
-
     val accountManager: AccountManager = AccountManager.get(context)
 
+    /**
+     * With DAVx5 4.3.3 address book accounts now contain the collection ID as a unique identifier.
```

Comment:
I'd replace "address book accounts" by "address book account names" here for clarity.